src/App.tsx

```diff
@@ -1,35 +1,56 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import Stopwatch from "./components/Stopwatch";
+import Add from "./assets/add.svg";
+import Delete from "./assets/delete.svg";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const prevStopwatches = JSON.parse(localStorage.getItem("ids") || "[]");
+  const prevCurrentId = parseInt(localStorage.getItem("currentId") || "0");
 
+  const [stopwatches, setStopwatches] =
+    useState<Array<number>>(prevStopwatches);
+  const [currentId, setCurrentId] = useState(prevCurrentId);
+
+  useEffect(() => {
+    localStorage.setItem("ids", JSON.stringify(stopwatches));
```

Comment:
If you want to keep stopwatches in localstorage when user leaves the page, you can use the cleanup function (https://react.dev/reference/react/useEffect#parameters). There were some problems with chrome (issue https://github.com/vercel/next.js/issues/23369), but developers say it doesn't reproduce anymore